tox.ini

```diff
@@ -4,22 +4,10 @@ envlist=
     py{37,38,39,310,311}-ethpm
     py{37,38,39,310,311}-core
     py{37,38,39,310,311}-integration-{goethereum,ethtester}
-    lint
+    py{38,39,310,311}-lint
+    py{37,38,39,310,311}-wheel-cli
     docs
     benchmark
-    py{37,38,39,310,311}-wheel-cli
-
-[isort]
```

Comment:
I'm curious about this change - is it better practice to have an `.isort.cfg` rather than have it in here?

Reply:
The recommended way per `isort`'s documentation is either a `.cfg` or within `pyproject.toml`. I figured this organizes it a bit better and we had settings split between `tox.ini` and `pyproject.toml`. I figured concentrating the configs to one place would be nice and when we move towards using `pyproject.toml` for the project setup I think that file will get a bit crowded. Open to any other idea though. Any thoughts on that?

Reply:
Nope, I think this is good. I was just curious if that was the recommended way.